Testscripts/Windows/PERF-SCHEDULER.ps1

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+param(
+    [String] $TestParams,
+    [object] $AllVMData,
+    [object] $CurrentTestData
+)
+
+function Main {
+    param (
+        $TestParams,
+        $AllVMData
+    )
+    # Create test result
+    $CurrentTestResult = Create-TestResultObject
+    $resultArr = @()
+
+    try {
+        $testName="scheduler"
+        $myString = @"
+./perf_scheduler.sh &> schedulerConsoleLogs.txt
+. utils.sh
+collect_VM_properties
+"@
+        Set-Content "$LogDir\Start${testName}Test.sh" $myString
+        Copy-RemoteFiles -uploadTo $AllVMData.PublicIP -port $AllVMData.SSHPort `
+            -files "$constantsFile,$LogDir\Start${testName}Test.sh" -username $user -password $password -upload
+        Copy-RemoteFiles -uploadTo $AllVMData.PublicIP -port $AllVMData.SSHPort `
+            -files $currentTestData.files -username $user -password $password -upload
+
+        Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -command "chmod +x *.sh" -runAsSudo | Out-Null
+        $testJob = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -command "./Start${testName}Test.sh" -RunInBackground -runAsSudo
+
+        while ((Get-Job -Id $testJob).State -eq "Running") {
```

Comment:
What is the exit condition of this while loop?

Reply:
The exit condition is that the $testJob is completed, which is the state of $testJob is not running.